package.json

```diff
@@ -46,7 +46,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-jest": "^24.9.0",
     "jest": "^24.9.0",
-    "magic-script-components": "git+https://github.com/magic-script/magic-script-components.git#XrClient",
+    "magic-script-components": "2.0.2",
```

Comment:
will you need to update this one version 3.0.0 comes out?

Reply:
Yes. I made it 3.0.0 first but then the unit tests failed since it can't pull that nonexistent version down

Reply:
I think part of our release exercise is going to be updating lots of internal version dependencies (especially in all the CLI templates)